doc/api/permissions.md

```diff
@@ -440,7 +446,151 @@ not adopt the origin of the `blob:` URL.
+using the [`--allow-child-process`][] and [`--allow-worker`][] respectively.
+
+#### Runtime API
+
+When enabling the Permission Model through the [`--experimental-permission`][]
+flag a new property `permission` is added to the `process` object.
+This property contains two functions:
+
+* `permission.deny(scope [,parameters])`
+
+API call to deny permissions at runtime ([`permission.deny()`][])
+
+```js
+process.permission.deny('fs'); // Deny permissions to ALL fs operations
+
+// Deny permissions to ALL FileSystemWrite operations
+process.permission.deny('fs.write');
+// deny FileSystemWrite permissions to the protected-folder
+process.permission.deny('fs.write', ['/home/rafaelgss/protected-folder']);
+// Deny permissions to ALL FileSystemRead operations
+process.permission.deny('fs.read');
+// deny FileSystemRead permissions to the protected-folder
+process.permission.deny('fs.read', ['/home/rafaelgss/protected-folder']);
+```
+
+API call to check permissions at runtime ([`permission.has()`][])
+
+```js
+process.permission.has('fs.write'); // true
+process.permission.has('fs.write', '/home/rafaelgss/protected-folder'); // true
+
+process.permission.deny('fs.write', '/home/rafaelgss/protected-folder');
+
+process.permission.has('fs.write'); // true
+process.permission.has('fs.write', '/home/rafaelgss/protected-folder'); // false
+```
+
+#### File System Permissions
+
+To allow access to the filesystem, use the [`--allow-fs-read`][] and
+[`--allow-fs-write`][] flags:
+
+```console
+$ node --experimental-permission --allow-fs-read=* --allow-fs-write=* index.js
+Hello world!
+(node:19836) ExperimentalWarning: Permission is an experimental feature
+(Use `node --trace-warnings ...` to show where the warning was created)
+```
+
+The valid arguments for both flags are:
+
+* `*` - To allow the all operations to given scope (read/write).
+* Paths delimited by comma (,) to manage reading/writing operations.
+
+Example:
+
+* `--allow-fs-read=*` - It will allow all `FileSystemRead` operations.
+* `--allow-fs-write=*` - It will allow all `FileSystemWrite` operations.
+* `--allow-fs-write=/tmp/` - It will allow `FileSystemWrite` access to the `/tmp/`
+  folder.
+* `--allow-fs-read=/tmp/,/home/.gitignore` - It allows `FileSystemRead` access
+  to the `/tmp/` folder **and** the `/home/.gitignore` path.
+
+Wildcards are supported too:
+
+* `--allow-fs-read:/home/test*` will allow read access to everything
+  that matches the wildcard. e.g: `/home/test/file1` or `/home/test2`
+
+There are constraints you need to know before using this system:
+
+* Native modules are not restricted by the Permission Model.
+* Relative paths are not supported through the CLI (`--allow-fs-*`).
+  The runtime API supports relative paths.
+* The model does not inherit to a child node process.
+* The model does not inherit to a worker thread.
+* Permission changes are not retroactively applied to existing resources.
+  Consider the following snippet:
```

Comment:
This doesn't belong to the bulleted list, does it?
```suggestion

Consider the following snippet:
```

Reply:
It should. It's a complement of the third bullet: `Permission changes are not retroactively applied to existing resources.`

Reply:
Fixed in 7dc06682d5